std_msgs/Header header
common/FloatTimeStamped cdcu_lvbat_volt
common/FloatTimeStamped cdcu_bms_batsoc
common/FloatTimeStamped cdcu_bms_batsoh
common/FloatTimeStamped cdcu_bms_batvolt
common/FloatTimeStamped cdcu_bms_batcurt
common/FloatTimeStamped cdcu_bms_battemp
common/Uint8TimeStamped cdcu_bms_chgst
common/Uint8TimeStamped cdcu_bms_dischgst
common/Uint8TimeStamped cdcu_bms_feedbackst